lib/editor/editor-manager.dart

```diff
@@ -15,7 +15,30 @@ class EditorManager {
     Map<String, dynamic> jsonMap = json.decode(data);
     AdventureData parsedData = AdventureData.fromJson(jsonMap);
     adventureData = parsedData;
-    adventureDataAsJson = data;
+  }
+
+  selectedParagraphAdventureDataJson() {
+    Paragraph paragraph = adventureData.paragraphs.firstWhere((paragraph) => paragraph.identifier == selectedParagraphId, orElse: () => Paragraph(identifier: selectedParagraphId, options: {}, text: '', type: ''));
+    return JsonEncoder.withIndent(' ').convert(paragraph.toJson());
+  }
+
+  String getAdventureDataAsJson() {
+    return JsonEncoder.withIndent(' ').convert(adventureData.toJson());
+  }
+
+  getParagraph(String id) {
+    Paragraph paragraph = adventureData.paragraphs.firstWhere((paragraph) => paragraph.identifier == id, orElse: () => Paragraph(identifier: selectedParagraphId, options: {}, text: '', type: ''));
```

Comment:
tworzenie Paragraph(identifier: selectedParagraphId, options: {}, text: '', type: '')
powtarza się może warto jakąś metodę na to dać ?